lambda-src/ecs_worker.py

```diff
@@ -0,0 +1,48 @@
+import os
+import logging
+import shlex
+
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+try:
+    import boto3
+    client = boto3.client('ecs')
+except ImportError:
+    logger.warning('boto3 is not installed. ECSTasks require boto3')
```

Comment:
You could just try to import `boto3` and let Python raise an exception which will exit the process anyway. I'm not that fussed but I'm not sure what the value of catching the exception is when we want to crash anyway.